Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'jbuilder', '~> 2.5'
 gem 'devise', '~> 4.3'
 gem 'devise_invitable', '~> 1.7.2'
 gem 'activeadmin', '~> 1'
-gem 'govuk_elements_rails'
+gem 'govuk_elements_rails', '~> 3.1'
```

Comment:
Could you separate that out to a different commit to explain the change?

Reply:
I like to limit gems to non-breaking subsets of versions with the use of `~>` to avoid anyone running `bundle update` and making breaking changes in our app. Ideally all our gems would do that.

http://bundler.io/v1.16/guides/updating_gems.html

Reply:
yes. Can you put it in a different commit though, explaining the above. Ideally doing it for all gems in one go? Just so we have a cleaner history of code changes.

Reply:
ok @maxf I'll do this in a separate PR.

Reply:
I removed the commit with the Gemfile changes.